packages/kbn-eslint-plugin-eslint/rules/no_restricted_paths.js

```diff
@@ -42,6 +42,23 @@ function isStaticRequire(node) {
   );
 }
 
+const winPathSeparatorRegExp = /\\/g;
+/**
+ * Converts win32 paths to posix format.
+ * @param {string} filePath
+ *
+ * @example
+ * toPosixPath('a\\b\\c') // 'a/b/c'
+ */
+function toPosixPath(filePath) {
+  return filePath.replace(winPathSeparatorRegExp, '/');
```

Comment:
Nit: I generally find `string.split('\\').join('/')` easier to understand and it's technically more performant too.